--- libb2-0.98.1/configure.ac.orig	2019-03-10 16:36:24.000000000 +0100
+++ libb2-0.98.1/configure.ac	2019-09-13 05:40:26.401162495 +0200
@@ -60,7 +60,7 @@
                [enable_native=yes]
 )
 
-AX_CHECK_COMPILE_FLAG([-O3], [CFLAGS=-O3])
+AX_CHECK_COMPILE_FLAG([-O3], [CFLAGS="$CFLAGS -O3"])
 dnl Not all architectures support -march=native
 if test $enable_native == "yes"; then
   AX_CHECK_COMPILE_FLAG([-march=native], [], [enable_native=no])
